src/mpyl/build.py

```diff
@@ -47,74 +46,71 @@ def run_mpyl(
     logger = logging.getLogger("mpyl")
     start_time = time.time()
     try:
-        run_result = RunResult(run_properties=run_properties, run_plan=run_plan)
-
-        if not run_result.has_projects_to_run(include_cached_projects=False):
-            logger.info("Nothing to do. Exiting..")
-            return run_result
-
-        logger.info("Run plan:")
-        console.print(Markdown(f"\n\n{run_result.status_line}  \n"))
-        run_plan.print_markdown(console, run_properties.stages)
-
-        try:
-            executor = Executor(
-                logger=logger,
-                run_properties=run_properties,
-                run_plan=run_plan,
-                steps_collection=StepsCollection(logger=logger),
-            )
-
-            run_result = run_build(
-                logger=logger, accumulator=run_result, executor=executor
-            )
-        except ValidationError as exc:
-            console.log(
-                f'Schema validation failed {exc.message} at `{".".join(map(str, exc.path))}`'
-            )
-            raise exc
-        except ExecutionException as exc:
-            run_result.exception = exc
-            console.log(f"Exception during build execution: {exc}")
-            console.print_exception()
+        run_plan = RunPlan.load_from_pickle_file()
+        console.print(Markdown("**Execution plan:**  \n"))
```

Comment:
Good catch! Yes, because on the current version in `master` that message doesn't show when [printing the run plan](https://github.com/Vandebron/gh-mpyl/blob/main/tests/reporting/formatting/test_resources/markdown_run_with_plan.md) at [the end of the step](https://github.com/Vandebron/gh-mpyl/blob/main/src/mpyl/build.py#L58).

We can easily make it do that, I just wanted to make sure the markdown console output stayed exactly the same before and after this PR 👍

Reply:
With the latest version of this PR I ended up being able to do this 👍